package.json

```diff
@@ -158,6 +162,11 @@
                 "codeAnalyzer.rules.category": {
                     "type": "string",
                     "description": "The categories of rules to run. Specify multiple values as a comma-separated list. Run 'sf scanner rule list -e' in the terminal for the list of categories associated with a specific engine."
+                },
+                "codeAnalyzer.apexGuru.enabled": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
Oh false by default. For some reason I thought we were going to just rely on the org perm check (making it async if it wasn't already) as the means to determine enablement by default. So I was expecting this to be true by default. So with this users will first need to enable the setting and then have their org (with perm enabled) authed.

Reply:
Yes, I have been going back and forth on this. But given how small the closed pilot is going to be, I don't want to initialize core service for rest of the users just to check the org perm. So, went ahead with this approach.